src/useLocalStore.ts

```diff
@@ -11,26 +11,13 @@ function wrapInTransaction(fn: Function) {
     }
 }
 
-export function useLocalStore<T>(initializer: (props?: any) => T, props?: any): T {
-    const [res] = useState(() => (props ? observable(props, {}, { deep: false }) : {}))
-
-    if (typeof props !== "undefined") {
-        if (process.env.NODE_ENV !== "production" && !isPlainObject(props)) {
-            throw new Error("useLocalStore expects an object as second argument")
+export function useLocalStore<T>(initializer: (props?: any) => T, current?: any): T {
+    const local = useState(() => {
```

Comment:
I think I've just preserved an convention that was here before: https://github.com/mobxjs/mobx-react-lite/blob/abdf573a343ce43cb5cbf7318202579f80990f35/src/useLocalStore.ts#L26

Reply:
Yea well, it kinda did make sense there because it was returning that value directly, but when you working with I think it makes more sense to destructure.